core/dataflow/src/test/java/edu/uci/ics/texera/dataflow/aggregator/AggregatorTest.java

```diff
@@ -0,0 +1,419 @@
+/**
+ *
+ */
+package edu.uci.ics.texera.dataflow.aggregator;
+
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import edu.uci.ics.texera.api.constants.test.TestConstants;
+import edu.uci.ics.texera.api.exception.TexeraException;
+import edu.uci.ics.texera.api.field.DateField;
+import edu.uci.ics.texera.api.field.DoubleField;
+import edu.uci.ics.texera.api.field.IField;
+import edu.uci.ics.texera.api.field.IntegerField;
+import edu.uci.ics.texera.api.field.StringField;
+import edu.uci.ics.texera.api.field.TextField;
+import edu.uci.ics.texera.api.schema.Attribute;
+import edu.uci.ics.texera.api.schema.AttributeType;
+import edu.uci.ics.texera.api.schema.Schema;
+import edu.uci.ics.texera.api.tuple.Tuple;
+import edu.uci.ics.texera.api.utils.TestUtils;
+import edu.uci.ics.texera.dataflow.source.scan.ScanBasedSourceOperator;
+import edu.uci.ics.texera.dataflow.source.scan.ScanSourcePredicate;
+import edu.uci.ics.texera.storage.DataWriter;
+import edu.uci.ics.texera.storage.RelationManager;
+import edu.uci.ics.texera.storage.constants.LuceneAnalyzerConstants;
+import junit.framework.Assert;
+
+/**
+ * @author avinash
+ * <p>
+ * This test uses the People_Table defined in TestConstants to verify the aggregator operator results
+ */
+public class AggregatorTest {
```

Comment:
Same comment for the `@author` tag position.

Regarding the bug we found when testing the GUI, in general I think it's good to add one test case for it. Can you remind me what kind of bug it is and why the current test cases are not able to cover it?

Reply:
The outputSchema being generated for Aggregator was null.